controller.py

```diff
@@ -151,6 +157,12 @@ def opds_catalog(self):
                 "title": "Remove items from one of your tracked collections.",
                 "templated": "true"
             },
+            {
+                "rel": "http://librarysimplified.org/rel/metadata/collection-metadata-requests",
+                "href": "/{collection_metadata_identifier}/metadata_requests",
+                "title": "Get items in your collection for which the metadata wrangler needs more information to process.",
+                "templated": "true"
+            },
```

Comment:
I would like to see a term like "missing" or "needs action" in here. With "requests" it's not clear who is making the request, and I find it confusing because the entity making the "request" for metadata is not the entity making the HTTP "request".